### DISCUSSION TITLE
请问下，要部署这个配置至少需要怎样的起步

### DISCUSSION THREAD

**Discussion by daguangmingtian** [xai-org/grok-1#229] [Q&A]
请问下，要部署这个配置至少需要怎样的起步，有知道的麻烦告知下，弄个清单最好，谢谢

**Comment by zhengqwe** [comment]
8个H100

**Comment by dextercai** [comment]
see #42

**Comment by chaoqiangl** [comment]
![image](https://github.com/xai-org/grok-1/assets/154698732/26a1dada-3c9c-4eb5-9ed8-571ebca2075e)
